drivers/driver_options_mock.go

```diff
@@ -0,0 +1,21 @@
+package drivers
```

Comment:
This was relied on by a couple modules and previously assumed to be in the same so it's been split out into its own exported file.